### DISCUSSION TITLE
Move to Architechuary Library

### DISCUSSION THREAD

**Discussion by Negative-light** [negative-light-media/soulsiphon#24] [Ideas]
This applies to all mods created by me. But should add the Architeechuary library?
This will allow us to build both forge and fabric mods using the same code base.

1. Pros - Cross-platform, single code base, fewer code changes on an update
2. Cons - not sure if architecture is supporting neo-forge, slower updates

**Comment by crazysmile11012** [comment]
you should but further testing would be needed

**Comment by Negative-light** [comment]
Ok had a discussion with @Healex2. I think we all agree this would be a good move so now to just do the research on what would be needed to add architecuary.
